plugins/scaffolder-backend-module-gitlab/api-report.md

```diff
@@ -0,0 +1,56 @@
+## API Report File for "@backstage/plugin-scaffolder-backend-module-gitlab"
```

Comment:
regenerate file with `yarn build:api-reports` after adjusting the documentation of the actions

Reply:
done